core/yolo/yolo_func.py

```diff
@@ -1,9 +1,12 @@
 from typing import Tuple
 from ultralytics import YOLO
+from PIL import Image
 
+Image.MAX_IMAGE_PIXELS = None
 
-def obj_det(img, model: YOLO, conf=0.25):
-    res = model.predict(source=img, save=False, save_txt=False, imgsz=640, conf=conf, device="cpu")
+
+def obj_det(img, model: YOLO, conf=0.25, img_size=(640, 640)):
```

Comment:
hvorfor giver vi en tuple som img_size her nu?

Reply:
Tried forcing yolo to return a square image. Works 'sometimes'.

Reply:
but why though?

Reply:
black bars = bad. check image in #101

Reply:
fixed in latets